docs/src/templates/pages/styleguide/ui-templates/page-components/cookie-banner.hbs

```diff
@@ -0,0 +1,361 @@
+---
+title: Component Library
+docs: false
+
+layout: site-frame
+styleguide: true
+---
+
+<div id="CookieAnchor"></div>
+  <main id="skipToMain" class="l-container" role="main">
+    <div class="g g--stacked g--alignCenter l-content">
```

Comment:
Would be good to abstract out the content below, as it's not really anything to do with the cookie policy banner (and if we change the structure of the content classes, it just gives us more to update).

I'd be tempted just to make this page a load of `p` tags inside an `l-container`, as there's not much need for any of the other content?